server/node/src/main/java/org/kaaproject/kaa/server/admin/servlet/ServletUtils.java

```diff
@@ -39,14 +43,32 @@ public static void prepareDisposition(HttpServletRequest request, HttpServletRes
         try {
             byte[] fileNameBytes = fileName.getBytes(isInternetExplorer ? "windows-1250" : "utf-8");
             String dispositionFileName = "";
-            for (byte b: fileNameBytes) {
-                dispositionFileName += (char)(b & 0xff);
+            for (byte b : fileNameBytes) {
+                dispositionFileName += (char) (b & 0xff);
             }
 
             String disposition = "attachment; filename=\"" + dispositionFileName + "\"";
             response.setHeader("Content-Disposition", disposition);
-        } catch(UnsupportedEncodingException ence) {
-           LOG.error("Exception catched: ", ence);
+        } catch (UnsupportedEncodingException ence) {
+            LOG.error("Exception catched: ", ence);
         }
-  }
+    }
+
+    public static String convertUuid(String json) {
```

Comment:
I assume that this is an attempt to hide a method that simply cleanup all uuids (set to null) using pretty method name. This is wrong. Never do this. Please investigate the root cause of the issue.